plugins/code-coverage-backend/src/service/router.ts

```diff
@@ -120,7 +119,7 @@ export const makeRouter = async (
    */
   router.get('/file-content', async (req, res) => {
     const { entity, path } = req.query;
-    const entityName = parseEntityName(entity as EntityRef);
+    const entityName = parseEntityRef(entity as string);
```

Comment:
Do we need the casting here? Would thought it wouldn't matter now as req.query is probably any? Maybe it's unknown though.

Reply:
Yeah unfortunately it comes out as `string | string[] | QueryString.ParsedQs | QueryString.ParsedQs[] | undefined` :D should have some kind of guard function that actually checks that it's nothing but a string, but that's for a later refactor